src/plot/series/custom-svg-series.js

```diff
@@ -99,7 +99,7 @@ class CustomSVGSeries extends AbstractSeries {
     if (animation) {
       return (
         <Animation {...this.props} animatedProps={ANIMATED_SERIES_PROPS}>
-          <CustomSVGSeries {...this.props} animation={null}/>
+          <CustomSVGSeries {...this.props} animation={false}/>
```

Comment:
I guess i don't understand how we would remove it from the parent to the child with out doing it this way?